### ISSUE TITLE
Listing of RFCs in the profile database

### ISSUE THREAD

**Issue by klensin** [ietf-tools/datatracker#4739]
### Describe the issue

Apparently, when RFCs on which an individual is listed as author are listed on that individual's profile page, only relatively recent documents are listed.  Unless there is an effort underway to get older RFCs and their authors into the database, perhaps the "RFC(s)" title line should read "RFC(s) after <date>" or equivalent.  Maybe similar treatment should be given to the listing of I-Ds, but that seems less important to me.

In trying to track this problem down, I discovered that 

* Jon Postel had no RFCs published before RFC 1111 in August 1989, which could lead a reader to believe that, e.g., RFCs 791 and 793.  Similarly Vint Cerf is credited only with RFC 1217, which might lead an inexperienced user of the system to conclude that he is good only for jokes.  Among the missing for both Jon and Vint are several current Internet Standards with normative references from recent documents, so that is not a trivial deal.

*  While, for example, Sandy Ginoza has her own page, listing 11 RFCs (but not 3700), attempting to look up Joyce Reynolds redirects to a page labeled "RFC Editor".  There, RFC 3700 is listed but the RFC shows Joyce and Sandy as co-editors.  The omission of 3700 in Sandy's listing looks like a bug.  Having no page at all for Joyce seems discourteous, especially since, like Jon, a number of important contributions, including documents still listed as Internet Standards, appear earlier.  

It appears to me that, even if it has to be done manually, we should have document pages in the datatracker for every document that represents a current Internet Standard (and maybe any current standards track document) and profile pages for their authors.  If that needs to be done manually, I feel sufficiently strongly about this that, if there is some reasonable tool that allow building such pages manually, I'd be willing to at least make a start on them.

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by rjsparks** [comment]
This will, indeed, require a manual effort to improve.
There isn't a simple tool available (at least at the moment) to facilitate helping with adjusting the data.

Further, there is a significant issue in the way the datatracker models RFCs - currently they are a type of internet-draft internally - a psuedo-version of a draft. This has proven to be an incorrect modeling mechanism - RFCs like 3700 did not come from an internet-draft. The current modeling would allow us to correct the author attributions for it (by adding documentauthor records). But we have a pending effort to change the models to make RFCs their own kind of document and at that point, a careful review of how authors are modeled will be needed, and is probably the best place to add in the effort to ensure that the attributions match what the document says. See #2917.

And then, we have another effort staged to make the subseries documents (BCP, STD, etc) their own kind of document. At the moment, the datatracker is actually _broken_ in modeling those things - it cannot reflect that a BCP consists of several RFCs. See #4426.

In summary, yes, there's a lot to do here, and I'm expecting to approach it as one coordinated project to get coverage, and I'll reach out to you for assistance when we get to the step of verifying the authors are correctly reflected.

In the meantime, if there's a particular RFC (such as those you call out) that you think are important enough to need to be adjusted to the extent we can adjust them now, send a note to support@ietf.org with the rfc number and the secretariat will look into adjusting the current documentauthor set.

**Comment by klensin** [comment]
Robert,
Thanks very much for the detailed response.  Given that we have lived without this for years, I don't see it as an urgent problem, so progressing in the way you suggest and solving interconnected problems together makes perfect sense to me.  If it is easy and it is possible to pick a date that works for everyone, it might still be worth patching the template for author profiles to say "RFC(s) after..." rather than "RFc(s)" to avoid being misleading, but, again, not a huge problem and, if it would require more than a few minutes work, it probably should be deferred.

thanks,
   john
